ci/install-bazel.sh

```diff
@@ -16,24 +16,26 @@
 
 set -eu
 
-if [[ $# -lt 1 ]]; then
-  echo "Usage: $0 <linux|osx>"
-fi
+readonly PLATFORM=$(uname -s | tr '[:upper:]' '[:lower:]')
+# Ensures that the platform is one that we expect
+grep -P "linux|darwin" <<<"${PLATFORM}" > /dev/null
 
-if [[ "$1" = "linux" ]]; then
-  readonly PLATFORM="linux-x86_64"
-elif [[ "$1" = "macos" ]] || [[ "$1" = "osx" ]]; then
-  readonly PLATFORM="darwin-x86_64"
-fi
+# Gets the latest version number of bazel from the GitHub JSON API.
+readonly BAZEL_VERSION=$(
+  curl -sL "https://api.github.com/repos/bazelbuild/bazel/releases/latest" \
+    | grep -P '"tag_name":' \
+    | cut -f4 -d\"
+)
+test -n "${BAZEL_VERSION}"
 
-readonly BAZEL_VERSION=0.20.0
 readonly GITHUB_DL="https://github.com/bazelbuild/bazel/releases/download"
-readonly SCRIPT_NAME="bazel-${BAZEL_VERSION}-installer-${PLATFORM}.sh"
+readonly SCRIPT_NAME="bazel-${BAZEL_VERSION}-installer-${PLATFORM}-x86_64.sh"
```

Comment:
$(uname -m)?

Reply:
I think that'd work. Although, bazel only has [x86_64 builds available](https://github.com/bazelbuild/bazel/releases), so do you think it's worth machine the machine hardware more dynamic like that?

Reply:
I don't know, but I'm assuming that if you're not on an x86_64 that you don't want to run the x86_64 installer.

Reply:
Actually, I think it is better w/ `uname -m`. Done.